src/content/docs/it/core-concepts/framework-components.mdx

```diff
@@ -0,0 +1,230 @@
+```
+
+:::caution
+Solo i componenti **Astro** (`.astro`) possono contenere componenti provenienti da più framework.
+:::
+
+## Passaggio di proprietà ai componenti del framework
+
+Puoi passare proprietà dai componenti Astro ai componenti del framework:
+
+```astro title="src/pages/frameworks-props.astro"
+---
+import TodoList from '../components/TodoList.jsx';
+import Counter from '../components/Counter.svelte';
+---
+<div>
+  <TodoList initialTodos={["learn Astro", "review PRs"]} />
+  <Counter startingCount={1} />
+</div>
+```
+
+:::caution[passare funzioni come proprietà]
+Puoi passare una funzione come proprietà a un componente del framework, ma funziona solo durante il rendering del server. Se provi a utilizzare la funzione in un componente idratato (ad esempio, come gestore eventi), si verificherà un errore.
+
+Questo perché le funzioni non possono essere _serializzate_ (trasferite dal server al client) da Astro.
+:::
+
+
+## Passaggio dei figli ai componenti del framework
+
+All'interno di un componente Astro, puoi **puoi** passare i figli ai componenti del framework. Ogni framework ha i propri modelli su come fare riferimento a questi bambini: React, Preact e Solid usano tutti un oggetto speciale chiamato `children`, mentre Svelte e Vue usano l'elemento `<slot />`.
+
+
+```astro title="src/pages/component-children.astro" {5}
+---
+import MyReactSidebar from '../components/MyReactSidebar.jsx';
+---
+<MyReactSidebar>
+  <p>Here is a sidebar with some text and a button.</p>
+</MyReactSidebar>
+```
+
+Inoltre, puoi utilizzare [slot nominati](/it/core-concepts/astro-components/#named-slots) per raggruppare insieme figli specifici.
+
+Per React, Preact e Solid questi slot verranno convertiti in un oggetto di livello superiore. I nomi degli slot che utilizzano `kebab-case` verranno convertiti in `camelCase`.
+
+```astro title="src/pages/named-slots.astro" /slot="(.*)"/
+---
+import MySidebar from '../components/MySidebar.jsx';
+---
+<MySidebar>
+  <h2 slot="title">Menu</h2>
+  <p>Here is a sidebar with some text and a button.</p>
+  <ul slot="social-links">
+    <li><a href="https://twitter.com/astrodotbuild">Twitter</a></li>
+    <li><a href="https://github.com/withastro">GitHub</a></li>
+  </ul>
+</MySidebar>
+```
+
+```jsx /{props.(title|socialLinks)}/
+// src/components/MySidebar.jsx
+export default function MySidebar(props) {
+  return (
+    <aside>
+      <header>{props.title}</header>
+      <main>{props.children}</main>
+      <footer>{props.socialLinks}</footer>
+    </aside>
+  )
+}
+```
+
+Per Svelte e Vue è possibile fare riferimento a questi slot utilizzando un elemento `<slot>` con l'attributo `name`. I nomi degli slot che utilizzano `kebab-case` verranno conservati.
+
+```jsx /slot name="(.*)"/
+// src/components/MySidebar.svelte
+<aside>
+  <header><slot name="title" /></header>
+  <main><slot /></main>
+  <footer><slot name="social-links" /></footer>
+</aside>
+```
+
+## Componenti della struttura inseriti l´uno nell'altro
```

Comment:
**nitpick**: I would not use the prime symbol `´` for the apostrophe, since we're using the basic straight apostrophe `'` everywhere else.